apps/meteor/tests/end-to-end/api/09-rooms.js

```diff
@@ -1707,4 +1707,131 @@ describe('[Rooms]', function () {
 				.end(done);
 		});
 	});
+
+	describe('/rooms.muteUser', () => {
+		let testChannel;
+
+		before('create a channel', async () => {
+			const result = await createRoom({ type: 'c', name: `channel.test.${Date.now()}-${Math.random()}` });
+			testChannel = result.body.channel;
+		});
+
```

Comment:
I think we should delete the resources after so there is no issue with the order of the tests

```javascript
after(() => deleteRoom({ type: 'c', roomId: testChannel._id }));
```